lisa/tools/strace.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+
+class Strace(Tool):
+    @property
+    def command(self) -> str:
+        return "strace"
+
+    def _check_exists(self) -> bool:
```

Comment:
Do we need to override this? The default in parent class should handle this case